src/components/__tests__/NotAuthorisedView.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import { fireEvent } from '@testing-library/react-native'
+import { mocked } from 'jest-mock'
+import React from 'react'
+import { Linking } from 'react-native'
+
+import { getLabels } from '../../services/helpers'
+import render from '../../testing/render'
+import NotAuthorisedView from '../NotAuthorisedView'
+
+jest.mock('react-native/Libraries/Linking/Linking', () => ({
+  openSettings: jest.fn(),
+}))
+
+describe('NotAuthorisedView', () => {
+  const setVisible = jest.fn()
+
+  it('should render description', () => {
+    const { getByText } = render(
+      <NotAuthorisedView setVisible={setVisible} description={getLabels().general.camera.noAuthorization.description} />
+    )
+    const isNoAuthDescription = getByText(getLabels().general.camera.noAuthorization.description)
```

Comment:
Sorry, I meant that you can set the variable before the render statement because you use `getLabels().general.camera.noAuthorization.description` twice. But it's also okay not to, it's only used twice here :)